src/vs/workbench/browser/dnd.ts

```diff
@@ -33,7 +33,7 @@ import { parse, stringify } from 'vs/base/common/marshalling';
 import { ILabelService } from 'vs/platform/label/common/label';
 import { hasWorkspaceFileExtension, isTemporaryWorkspace, IWorkspaceContextService } from 'vs/platform/workspace/common/workspace';
 import { withNullAsUndefined } from 'vs/base/common/types';
-import { ITreeDataTransfer } from 'vs/workbench/common/views';
+import { IDataTransfer } from 'vs/workbench/common/dataTransfer';
```

Comment:
Since we already have `dnd.ts` in `browser` I would suggest to rename `common/dataTransfer.ts` to `common/dnd.ts` and maybe we can move even more non-browser related things over later.